tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -196,6 +196,24 @@ For a complete list of built-in GLSL functions see the :ref:`Shading Language do
 | vec_type **fwidth** ( vec_type )                                       |                                                  |
 +------------------------------------------------------------------------+--------------------------------------------------+
 
+**textureSize() workaround**
+
+OpenGL ES2.0 do not support ``TextureSize()``. If you need to get the size of a texture you need to do it in the old 
```

Comment:
OpenGL ES2.0 -> GLES2
do -> does
TextureSize -> textureSize
"If you need to get..." -> "You can get the size of a texture the old fashioned way by passing in..."

Reply:
Thanks :)